sklearn/df_clsf.py

```diff
@@ -40,9 +41,11 @@
     except ImportError:
         from sklearn.ensemble import RandomForestClassifier
 
-# Load data
-X = np.load(params.filex.name)
-y = np.load(params.filey.name)
+# Load and convert data
+X = convert_data(np.load(params.filex.name),
+    params.dtype, params.data_order, params.data_format)
```

Comment:
`params.dtype` argument should be indented to match the level of `np.load`. See https://www.python.org/dev/peps/pep-0008/#indentation
I would recommend using `flake8` to check for these things.

Also, because `convert_data` has kwargs, I would prefer for them to be passed as keyword arguments rather than positional arguments.